testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/TermsAndConditionsTest.java

```diff
@@ -186,14 +186,8 @@ public void termsDeclinedAccount() {
             assertNull("expected null for terms acceptance user attribute " + TermsAndConditions.USER_ATTRIBUTE,
                     attributes.get(TermsAndConditions.USER_ATTRIBUTE));
         }
-        assertThat(DroneUtils.getCurrentDriver().getTitle(), equalTo("Account Management"));
```

Comment:
I did not get how the `state` param is now forcing us to remove these assertions. The flow is the same as when not setting the `state` parameter, and the `error` (and error_description) parameter should cause the account to be rendered with the expected user message?